### PR TITLE
Testing/test for module table

### PR DESCRIPTION
**Pull request by BUGBOUNTYchrisg8691**
# What's here (& why)

- (include link(s) to applicable Trello card(s))

# Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [x] Incomplete / work-in-progress, PR is for discussion or feedback
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How has this been tested?

- [x] Tested locally
- [ ] Test suite provides the same results as prior to making changes

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: src/components/RestCourses/CourseCard.js [hunk 1]
```diff
@@ -1,27 +1,27 @@
 import React from 'react';
 import 'antd/dist/antd.css';
-import { Card } from 'antd';
+// import { Card } from 'antd';
```
Comment (chazkiker2) [inline src/components/RestCourses/CourseCard.js:3]: delete commented-out code

File: src/components/RestCourses/CourseCard.js [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
-    >
-      <h4>
-        <strong>Course Code:</strong> {course.coursecode}
-      </h4>
-      <p>
-        <strong>Description:</strong> {course.coursedescription}
-      </p>
-      {props.children}
-    </Card>
+    <Collapse accordian className="course-card">
```
Comment (chazkiker2) [inline src/components/RestCourses/CourseCard.js]: accordion time baby

File: src/components/RestModules/EditModuleForm.js [hunk 3]
```diff
@@ -79,10 +79,21 @@ function EditModuleForm(props) {
...
+            maxLength={250}
+            id="modulecontent"
+            name="modulecontent"
+            value={values.modulecontent}
+            onChange={changeValues}
+            rows={25}
           />
           <div style={{ color: 'red' }}>
-            {errors.moduledescription ? `${errors.moduledescription}` : ''}
+            {errors.modulecontent ? `${errors.modulecontent}` : ''}
```
Comment (chazkiker2) [inline src/components/RestModules/EditModuleForm.js:96]: could replace with `{errors.modulecontent ?? ""}`

File: src/components/RestModules/ModuleCard.js [hunk 4]
```diff
@@ -1,15 +1,14 @@
...
-  const { module } = props;
   return (
     <>
-      <Card title={module.modulename}>
-        <h3>Description: {module.moduledescription}</h3>
-        {/* PLEASE DO NOT REMOVE THE FOLLOWING LINE */}
-        {props.children}
-      </Card>
+      {/* <Card title={module.modulename}>
+        <h3>Description: {module.moduledescription}</h3> */}
```
Comment (chazkiker2) [inline src/components/RestModules/ModuleCard.js:7]: delete commented out code

File: src/components/RestModules/ModuleCard.js [hunk 5]
```diff
@@ -1,15 +1,14 @@
...
+      {/* PLEASE DO NOT REMOVE THE FOLLOWING LINE */}
+      {props.children}
+      {/* </Card> */}
```
Comment (chazkiker2) [inline src/components/RestModules/ModuleCard.js:10]: delete commented out code

File: src/components/RestModules/ModuleCard.js [hunk 6]
```diff
@@ -1,15 +1,14 @@
...
     <>
-      <Card title={module.modulename}>
-        <h3>Description: {module.moduledescription}</h3>
-        {/* PLEASE DO NOT REMOVE THE FOLLOWING LINE */}
-        {props.children}
-      </Card>
+      {/* <Card title={module.modulename}>
+        <h3>Description: {module.moduledescription}</h3> */}
+      {/* PLEASE DO NOT REMOVE THE FOLLOWING LINE */}
+      {props.children}
```
Comment (chazkiker2) [inline src/components/RestModules/ModuleCard.js:9]: and if this component now ONLY renders `{props.children}` in a `<React.Fragment />` then it should ~~probably~~ definitely not be a component anymore. just taking up space for no reason

File: src/components/RestModules/ModulesTable.js [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+      dataIndex: 'actions',
+      render: (text, record) => {
+        return (
+          <Space size="middle">
+            {record._links.self.href && (
+              <EditOutlined
+                key="edit"
+                onClick={e => {
+                  e.preventDefault();
+                  console.log('edit module', record._links.self.href);
```
Comment (chazkiker2) [inline src/components/RestModules/ModulesTable.js:47]: remove `console.log`

File: src/components/RestModules/ModulesTable.js [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+                }}
+              />
+            )}
+            {record._links.self.href && (
+              <DeleteOutline
+                key="delete"
+                onClick={e => {
+                  e.preventDefault();
+                  console.log('delete module', record._links.self.href);
```
Comment (chazkiker2) [inline src/components/RestModules/ModulesTable.js:56]: remove `console.log`

**Review by chazkiker2 — CHANGES_REQUESTED** [review]
looks good for the most part, just a couple of changes left in comment. 

also, it looks like you failed one of the checks. Did you by chance get rid of `.env.sample` at some point?? if so, add that file back in to pass the failing check

**Comment by chazkiker2** [comment]
looks like the only change you made was changing that "accordion" prop... Can you add `.env.sample` back into your project & delete commented-out code??

**Review by sebass482 — COMMENTED** [review]
Looks good Chris. There is a check failing and Chaz pointed out some stuff so I think there are more commits for this branch? Lmk

**Pull request closed without merging**